src/main/java/com/znsio/e2e/tools/Drivers.java

```diff
@@ -377,6 +377,7 @@ private WebDriver createChromeDriver (String forUserPersona,
         prefs.put("profile.default_content_setting_values.media_stream_mic", 1);
         prefs.put("profile.default_content_setting_values.media_stream_camera", 1);
         prefs.put("protocol_handler.excluded_schemes", excludedSchemes);
+        prefs.put("safebrowsing.enabled", "true");
```

Comment:
Why is this change only for ChromeDriver?

Reply:
since we are using the Chrome browser, we checked this option locally and it worked. If needed, we can add it in other browsers' options as well. Not sure how to test those browsers then

Reply:
do the other browsers have similar option? If yes, lets add that

Reply:
that didn't work out; hence i removed it